frontend/src/component/Search/AdvancedSearchSpaceList.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import {
+  CONTENT_TYPE,
+  Icon,
+  ListItemWrapper,
+  PAGE
+} from 'tracim_frontend_lib'
+import { Link } from 'react-router-dom'
+// import PropTypes from 'prop-types'
+
+require('./AdvancedSearchSpaceList.styl')
+
+// TODO - G.B. - 2021-02-16 - This component should be updated at https://github.com/tracim/tracim/issues/4097
+
+export const AdvancedSearchSpaceList = props => {
```

Comment:
I would normally ask to group AdvancedSearchSpaceList and AdvancedSearchContentList since they have the same code, but if I understand correctly, It will be rewritten in #4097, right?

Reply:
This component will be update at #4180, it's not the final version but they are not the same.